aws/config.ts

```diff
@@ -11,80 +11,146 @@ function getBooleanOrDefault(key: string, def: boolean): boolean {
   return ret == undefined ? def : ret;
 }
 
-export const LightbendNamespace = "lightbend";
-export const AwsOTelCollectorNamespace = "aws-otel-collector";
-
-export const eksVpcArgs: awsx.ec2.VpcArgs = {
-  numberOfAvailabilityZones: config.getNumber("vpc.numberOfAvailabilityZones") || 2,
-};
+class Namespaces {
+  static readonly LightbendNamespace: string = "lightbend";
+  static readonly AwsOTelCollectorNamespace: string = "aws-otel-collector";
+}
 
-export const eksClusterOptions: eks.ClusterOptions = {
+class Versions {
```

Comment:
Not specifically about this line, but this file as a whole: configuration was getting a little messy, with defaults, versions, configurations, and methods all mixed. I'm still keeping everything in a single file, but we can have some organization for all those configs.